steps/configuration.json

```diff
@@ -2,5 +2,6 @@
   "s3_corporate_bucket": "${s3_corporate_bucket}",
   "s3_published_bucket": "${s3_published_bucket}",
   "dks_decrypt_endpoint": "${dks_decrypt_endpoint}",
-  "dks_max_retries": "${dks_max_retries}"
+  "dks_max_retries": "${dks_max_retries}",
+  "extra_python_files": ${extra_python_files}
 }
```

Comment:
it doesn't like this - maybe missing quotes?

Reply:
Github doesn't like that because it is not valid json. But the variable "extra_python_files" gets interpolated into an array, that is why we don't enclosed it with quotes here

Reply:
Ooh that makes sense